crates/ruff_linter/src/rules/pydocstyle/helpers.rs

```diff
@@ -61,3 +66,63 @@ pub(crate) fn should_ignore_definition(
             })
     })
 }
+
+pub(crate) fn get_section_contexts<'a>(
+    docstring: &'a Docstring<'a>,
+    convention: Option<&'a Convention>,
+) -> SectionContexts<'a> {
+    let section_contexts;
+    match convention {
+        Some(Convention::Google) => {
+            section_contexts = SectionContexts::from_docstring(docstring, SectionStyle::Google);
+        }
+        Some(Convention::Numpy) => {
+            section_contexts = SectionContexts::from_docstring(docstring, SectionStyle::Numpy);
+        }
+        Some(Convention::Pep257) | None => 'unspecified: {
+            // There are some overlapping section names, between the Google and NumPy conventions
+            // (e.g., "Returns", "Raises"). Break ties by checking for the presence of some of the
+            // section names that are unique to each convention.
+
+            // If the docstring contains `Parameters:` or `Other Parameters:`, use the NumPy
+            // convention.
+            let numpy_sections = SectionContexts::from_docstring(docstring, SectionStyle::Numpy);
+            if numpy_sections.iter().any(|context| {
+                matches!(
+                    context.kind(),
+                    SectionKind::Parameters
+                        | SectionKind::OtherParams
+                        | SectionKind::OtherParameters
+                )
+            }) {
+                section_contexts = numpy_sections;
+                break 'unspecified;
```

Comment:
I'd probably replace this `break` control flow with an early `return`, since this method just returns `section_contexts` anyway.